src/pyscaffold/templates/__init__.py

```diff
@@ -14,6 +14,18 @@
 from .. import dependencies as deps
 from .. import toml
 
+if sys.version_info[:2] >= (3, 7):
+    from importlib.resources import read_text
+else:
+    from pkgutil import get_data
+
+    def read_text(package, resource, encoding="utf-8") -> str:
+        data = get_data(package, resource)
+        if data is None:
+            raise FileNotFoundError(f"{resource!r} resource not found in {package!r}")
+        return data.decode(encoding)
```

Comment:
Instead of doing that I could also add `importlib_resources` as a runtime dependency... but I felt like that was unnecessary for just one function that can be done with `pkgutil`.

Reply:
Yeah, makes sense.